db/seeds.rb

```diff
@@ -0,0 +1,157 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+require 'json'
+
+# distance of each planet in our solar system from Earth in Astronomical units"
+$distances = { 
+	"mercury" => 0.39,
+	"venus" => 0.72,
+	"earth" => 1.00,
+	"mars" => 1.52,
+	"87 sylvia" => 2.87,
+	"243 ida" =>  3.25,
+	"216 kleopatra" => 3.56, 
+	"50000 quaoar" => 4.12,
+	"136472 makemake" => 4.12,
+	"45 eugenia" => 4.2,
+	"jupiter" => 5.20,
+	"saturn" =>9.58,
+	"uranus" => 19.20,
+	"90482 orcus" => 29.21,
+	"neptune" => 30.05,
+	"pluto" => 39.48,
+	"136108 haumea" => 49.75,
+	"136199 eris" => 96.56,
+}
+
+$translations = {
+	"mercure" => "mercury",
+	"vénus" => "venus",
+	"terre" => "earth",
+	"mars" => "mars",
+	"jupiter" => "jupiter",
+	"saturne" => "saturn",
+	"uranus" => "uranus",
+	"neptune" => "neptune",
+	"pluton" => "pluto",
+	"haumea" => "136108 haumea",
+	"eris" => "136199 eris",
+	"eugenia" => "45 eugenia",
+	"sylvia" => "87 sylvia",
+	"orcus" => "90482 orcus",
+	"ida" => "243 ida",
+	"kleopatra" => "216 kleopatra",
+	"quaoar" => "50000 quaoar",
+	"makemake" => "136472 makemake"
+}
+
+# To interpolate into such a string: "The atmospheric conditions on #{planet.name} are #{} "
+$weather = [
+	"extremely hot",
+	"below freezing",
+	"mild",
+	"clear",
+	"mostly sunny",
+	"cloudy",
+	"windy",
+	"extremely windy",
+	"dangerously unpredictable",
+	"very wet",
+	"scorchingly hot",
+	"muggy",
+	"steamy",
+	"arid",
+	"insufferably cold",
+	"beautiful",
+	"bright and sunny",
+	"mostly cloudless",
+	"gentle",
+	"gloomy",
+	"bitterly cold",
+	"devastatingly cold",
+	"unforgiving",
+	"icy",
+	"unrelenting",
+	"harsh",
+	"thunderous",
+	"blizzardous",
+	"crystallised by frost",
+	"foggy"
+]
+
+planets_url = open("https://api.le-systeme-solaire.net/rest/bodies/").read
+$planets_json = JSON.parse(planets_url)
```

Comment:
Would include these lines in a method. The seed file will grow more with additional models and these lines will float around without a connection

Reply:
i thought about that, but if i included these lines in a method (i.e. get_json) and called get_json from the body_builder method, then every time the body_builder runs it would make a call to the API for the same info. I am happy not using a global variable but I wouldn't otherwise know how to access a json (declared only once outside body_builder) from inside body_builder. do you have any ideas? 😕